main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,258 @@
+
+## Creating and sending a fungible asset
+
+In ERTP, digital assets are created by a `mint`. You must have
+access to a `mint` object to create new assets of the `mint` object's
+associated `brand`. 
+
+For instance, let's say you want to create a new community
+currency called 'BaytownBucks'. You start by creating or 
+getting access to a `mint` object whose associated brand is 'BaytownBucks'.
+```js
+import { makeMint } from '@agoric/ertp';
+const baytownBucksMint = makeMint('BaytownBucks');
+```
+
+Now, use the `mint` to create 1000 new BaytownBucks.
+
+```js
+const { issuer, mint } = makeIssuerKit('BaytownBucks');
+const bb1000 = amountMath.make(1000);
+const newPayment = mint.mintPayment(bb1000);
+const bbpurse = issuer.makeEmptyPurse();
+const bb1000Amount = bbpurse.deposit(newPayment);
+```
+
+The act of minting created 1000 BaytownBucks and stored them together in a
+`payment`. We then deposited the `payment` into a `purse` for BaytownBucks.
+
+Now, let's distribute the BaytownBucks to others. To send assets
+in ERTP, we withdraw a `payment` from a `purse`.
+
+```js
+const paymentForAlice = purse.withdraw(bucks10);
+```
+
+Like our `purse`, this `payment` contains BaytownBucks, but unlike purses,
+we use payments to represent tokens in transit. A `payment` can be
+sent to someone else, a `purse` should never be sent to someone else.
+
+Now let's send the `payment` to Alice.
+
+Before she can receive a `payment` in BaytownBucks, Alice needs to 
+create an empty `purse` for BaytownBucks and otherwise get ready to
+receive payments. 
+
+For this to happen, Alice needs access to `baytownBucksIssuer()`.
+However, she does not need access to `baytownBucksMint()`.
+If she had access to the `mint()`, she could create BaytownBucks herself
+by calling `baytownBucksMint.mint()`. 
+
+```js
+const aliceBaytownBucksPurse = baytownBucksIssuer.makeEmptyPurse()
+aliceBaytownBucksPurse.getAmount(); // Empty, so the value is 0
+```
+Next, we make a deposit facet for Alice's `purse`. A deposit facet is an object 
+associated with a `purse` which only allows deposits to its `purse` and cannot be used
+to make a withdrawal from its `purse`.
+```js
+const aliceDepositFacet = purse.makeDepositFacet();
+```
+Here `aliceDepositFacet` is an object reference, and we can call its `receive` method
+to deposit this `payment` into its associated `purse`. 
+```js
+aliceDepositFacet.receive(paymentForAlice);
+```
+
+And here is a more complete example using Agoric's Board, a simple "bulletin board"
+for posting and getting Id-value pairs:
+```js
+// Alice posts an Id-value pair to the Board, where the value is the deposit facet object
+// for her `purse`.
+// Alice then gives us the boardId to use for the transfer: aliceDepositFacetBoardId
+// We retrieve the value from the boardID and then deposit the payment in the deposit facet.
```

Comment:
thanks for adding this! I think it will be very helpful for our developers in our current system